### PR TITLE
add process status to each output line

### PR DESCRIPTION
**Pull request by xibuka**
when you have a lot of files, it's kind to let the user know how many we have processed.
So add this information to each line.

### CODE REVIEW

**Comment by ivandokov** [comment]
Thank you for the PR. 
Can you please add the progress to the xmp files too as it looks weird now.

image

Also please add a spacing after the closing bracket to separate it from the file path like this:
`[1/234] /Users/...`

File: src/phockup.py [hunk 1]
```diff
@@ -35,6 +35,13 @@ def __init__(self, input, output, **args):
         self.dry_run = args.get('dry_run', False)
 
         self.check_directories()
+
+        total_files = 0
+        for _, _, filenames in os.walk(input):
+            total_files += len([f for f in filenames if f not in ignored_files])
+        self.total_files=total_files
```
Comment (ivandokov) [inline src/phockup.py:42]: Can you please add spacing around `=`

**Comment by xibuka** [comment]
Fixed the spacing issue but stuck at `XMP` file.

The output for `XMP` is different between the 1st run and 2nd run.
For instance, we have 2 files `aaa.nef` and `aaa.xmp`. 
The 1st run will show both, but the 2nd run will only show `aaa.nef` as duplicated and `aaa.xmp` will not be checked at all.
Check my below test log, the final status of 2nd run is wired. 
Any ideas?

```
# first run
$ sudo phockup pic1 pic2 
[1/18] pic1/IMGP2490.JPG => pic2/2011/08/21/20110821-201559.jpg
[2/18] pic1/IMGP2491.JPG => pic2/2011/08/21/20110821-201614.jpg
[3/18] pic1/IMGP2492.JPG => pic2/2011/08/21/20110821-201622.jpg
[4/18] pic1/IMGP2493.JPG => pic2/2011/08/21/20110821-201632.jpg
[5/18] pic1/IMGP2494.JPG => pic2/2011/08/21/20110821-201706.jpg
[6/18] pic1/IMGP2496.JPG => pic2/2011/08/21/20110821-201732.jpg
[7/18] pic1/IMGP2497.JPG => pic2/2011/08/21/20110821-201743.jpg
[8/18] pic1/IMGP2499.JPG => pic2/2011/08/21/20110821-201814.jpg
[9/18] pic1/IMGP2500.JPG => pic2/2011/08/21/20110821-201820.jpg
[10/18] pic1/IMG_2239.JPG => pic2/2017/08/13/20170813-215802214午後.jpg
[11/18] pic1/pdfx-xmp-example.nef => pic2/unknown/pdfx-xmp-example.nef
[12/18] pic1/pdfx-xmp-example.xmp => pic2/unknown/pdfx-xmp-example.xmp
[13/18] pic1/VIDEO_TS/VIDEO_TS.BUP => pic2/unknown/VIDEO_TS.BUP
[14/18] pic1/VIDEO_TS/VIDEO_TS.IFO => pic2/unknown/VIDEO_TS.IFO
[15/18] pic1/VIDEO_TS/VIDEO_TS.VOB => pic2/unknown/video_ts.vob
[16/18] pic1/VIDEO_TS/VTS_01_0.BUP => pic2/unknown/VTS_01_0.BUP
[17/18] pic1/VIDEO_TS/VTS_01_0.IFO => pic2/unknown/VTS_01_0.IFO
[18/18] pic1/VIDEO_TS/VTS_01_1.VOB => pic2/unknown/vts_01_1.vob

# 2nd run
$ sudo phockup pic1 pic2 
[1/18] pic1/IMGP2490.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201559.jpg
[2/18] pic1/IMGP2491.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201614.jpg
[3/18] pic1/IMGP2492.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201622.jpg
[4/18] pic1/IMGP2493.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201632.jpg
[5/18] pic1/IMGP2494.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201706.jpg
[6/18] pic1/IMGP2496.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201732.jpg
[7/18] pic1/IMGP2497.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201743.jpg
[8/18] pic1/IMGP2499.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201814.jpg
[9/18] pic1/IMGP2500.JPG => skipped, duplicated file pic2/2011/08/21/20110821-201820.jpg
[10/18] pic1/IMG_2239.JPG => skipped, duplicated file pic2/2017/08/13/20170813-215802214午後.jpg
[11/18] pic1/pdfx-xmp-example.nef => skipped, duplicated file pic2/unknown/pdfx-xmp-example.nef
[12/18] pic1/VIDEO_TS/VIDEO_TS.BUP => skipped, duplicated file pic2/unknown/VIDEO_TS.BUP
[13/18] pic1/VIDEO_TS/VIDEO_TS.IFO => skipped, duplicated file pic2/unknown/VIDEO_TS.IFO
[14/18] pic1/VIDEO_TS/VIDEO_TS.VOB => skipped, duplicated file pic2/unknown/video_ts.vob
[15/18] pic1/VIDEO_TS/VTS_01_0.BUP => skipped, duplicated file pic2/unknown/VTS_01_0.BUP
[16/18] pic1/VIDEO_TS/VTS_01_0.IFO => skipped, duplicated file pic2/unknown/VTS_01_0.IFO
[17/18] pic1/VIDEO_TS/VTS_01_1.VOB => skipped, duplicated file pic2/unknown/vts_01_1.vob
```

**Comment by ivandokov** [comment]
That's indeed an issue as the xmp files are processed only if the media file is not a duplicate.

**Pull request closed without merging**